explorer/client/conversion-scripts/move-json-converter.js

```diff
@@ -0,0 +1,7 @@
+const fs = require('fs');
```

Comment:
Do we need to commit this folder `explorer/client/conversion-scripts`?

Reply:
that's a good question, i think @Andrew47 might know more about this than me